test/lit.cfg.py

```diff
@@ -79,4 +79,7 @@
   config.available_features.add('llhd-sim')
   tools.append('llhd-sim')
 
+if 'Windows' not in config.host_os:
```

Comment:
Nit: `not-windows` is a little different re:features as it's negative not positive-- could we have a "windows" feature and mark these tests `UNSUPPORTED: windows`?

Reply:
Actually, I think `system-windows` is already handled by lit?  If so, we can use that and just need to add `UNSUPPORTED: system-windows` to the relevant files.

Reply:
Changed to `UNSUPPORTED: system-windows` - thanks!